$OpenBSD: patch-sql_sql_parse_cc,v 1.13 2013/10/04 07:09:18 giovanni Exp $

Fix floating point string printing in MySQL status output
(MySQL Bug #61205)

--- sql/sql_parse.cc.orig	Mon Sep  9 21:23:26 2013
+++ sql/sql_parse.cc	Thu Oct  3 15:23:29 2013
@@ -1593,7 +1593,7 @@ bool dispatch_command(enum enum_server_command command
     length= my_snprintf(buff, buff_len - 1,
                         "Uptime: %lu  Threads: %d  Questions: %lu  "
                         "Slow queries: %lu  Opens: %lu  Flush tables: %lu  "
-                        "Open tables: %u  Queries per second avg: %u.%u",
+                        "Open tables: %u  Queries per second avg: %u.%03u",
                         uptime,
                         (int) thread_count, (ulong) thd->query_id,
                         current_global_status_var.long_query_count,
